docs/documentation/upgrading/topics/migrate_db.adoc

```diff
@@ -28,6 +33,8 @@ default `connections-jpa` provider:
 kc.[sh|bat] start --spi-connections-jpa-quarkus-migration-strategy=manual
 ----
 
+This `start` command creates the SQL file with the commands you need to run.
+
 When you start the server with this configuration, the server checks if the database needs to be migrated.
 The required changes are written to the `bin/keycloak-database-update.sql` SQL file that you can review and manually run against the database.
```

Comment:
These two paragraphs seem to explain the same thing, with the second being more detailed. If this is intended, then we can keep it this way. 

Otherwise I would suggest to either remove the blank line between them to keep it in the same paragraph, or combining the two paragraphs even more.